packages/components/src/color-palette/index.js

```diff
@@ -29,12 +30,21 @@ export default function ColorPalette( {
 			<CircularOptionPicker.Option
 				key={ color }
 				isSelected={ value === color }
+				selectedIconProps={
+					value === color
+						? {
+								fill: tinycolor
+									.mostReadable( color, [ '#111', '#fff' ] )
```

Comment:
Should we use `#000` instead? cc @jasmussen

Reply:
We should. How much more black could it be? And the answer is none. None more black.

Reply:
👍 Updated.